Pipfile

```diff
@@ -16,6 +16,7 @@ chromedriver-installer = "*"
 testfixtures = "*"
 sqlalchemy = "==1.2.2"
 "psycopg2" = "==2.7.3.2"
+retrying = "*"
```

Comment:
We should pin this (despite the low maintenance of the project).

Reply:
Do we need to pin any of these, that's what the pipfile.lock does right?